projects/Mallard/src/components/issue/issue-row.tsx

```diff
@@ -165,9 +166,35 @@ const IssueButtonContainer = React.memo(
     ),
 )
 
+/**
+ * Custom palette for Front titles. We use the dark variants for some pillars
+ * because their "main" counterpart isn't legible enough for text on a light
+ * background.
+ */
+const getCustomColor = (appr: Appearance): colour => {
```

Comment:
Could we use an object instead of a nested switch? Would be easier to read.

Reply:
yes!

Reply:
Let me know when you have done it.